Dockerfile

```
@@ -1,4 +1,4 @@
-FROM node:12.19.0-alpine
+FROM node:12.21.0-alpine
```

Hey Nick thanks for bringing this up! I think we can straight-up use `12-alpine`. That way we don't need to care about minor updates in the future.

```suggestion
FROM node:12-alpine
```
I'm also open to update node to 14, but that might be a little bit of a bigger task: checking deps, checking used functions in different secret store implementations etc..